Source/WebInspectorUI/UserInterface/Views/ConsoleSourceMapMessageGroup.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2023 Apple Inc.  All rights reserved.
+ * Copyright (C) 2023 Jonathan Davis
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1.  Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ * 2.  Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ * 3.  Neither the name of Apple Inc. ("Apple") nor the names of
+ *     its contributors may be used to endorse or promote products derived
+ *     from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY APPLE AND ITS CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL APPLE OR ITS CONTRIBUTORS BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+ * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+WI.ConsoleSourceMapMessageGroup = class ConsoleSourceMapMessageGroup extends WI.ConsoleGroup
+{
+    constructor(parentGroup)
+    {
+        super();
+
+        this._messageCount = 0;
+        this._titleElement = null;
+    }
+    
+    // Public
+    
+    render()
+    {
+        var groupElement = document.createElement("div");
+        groupElement.classList.add("console-sourcemap-errors-group");
+        groupElement.group = this;
+        this.element = groupElement;
+
+        // TODO set consoleMessage to use WI.UIStringWI.UIString("Source Map loading errors")
+        let consoleMessage = new WI.ConsoleMessage(WI.mainTarget, WI.ConsoleMessage.MessageSource.Other, WI.ConsoleMessage.MessageLevel.Error, WI.UIString("Source Map loading errors"));
+
+        let consoleMessageView = new WI.ConsoleMessageView(consoleMessage);
+        consoleMessageView.render();
+
+        var titleElement = consoleMessageView.element;
+        this._titleElement = titleElement;
+        this._titleElement.classList.add(WI.LogContentView.ItemWrapperStyleClassName, "hidden");
+        this._titleElement._expandable = false;
```

Comment:
I commented with an alternative to always keep the group collapsible / expandable. 
But if we _needed_ to keep this state, it should live on a member of class not on a DOM element.

Reply:
Addressed in the new approach.